gulpfile.js

```diff
@@ -34,13 +34,22 @@ const IMAGES_DIR = 'build/images';
 function parseCommandLine() {
   commander
     .option(
-      '--grep [pattern]',
+      '--grep [pattern...]',
       'Run only tests where filename matches a pattern'
     )
+    .option('--watch', 'Continuously run tests (default: false)', false)
+    .option(
+      '--browser [browsers...]',
+      'Run test with this browsers (default "ChromeHeadless_Custom")',
+      ['ChromeHeadless_Custom']
```

Comment:
Unfortunately with this option you can only specify `ChromeHeadless` or `Chrome` since those are the only launchers installed. What I was going to suggest doing instead is to have an option that doesn't launch any browser automatically but instead just launches Karma and waits for the user to load the Karma server in a browser of their choice. When calling the Karma API, this just requires setting the `browsers` argument to `null`.

I'm sure what the best name for such an option would be - `--no-launch-chrome` ("Don't launch headless Chrome to run the tests. Navigate to http://localhost:9876/ to run the tests.")?

Reply:
I think it is a very good idea to allow developers to run the test in whatever browser they wish. I added the option `--no-browser` for that.